tests/core/test_core_cli.py

```diff
@@ -165,14 +165,14 @@ def test_dev_mode_clean_all_entities_of_the_latest_version():
     # Assert number of entities of the latest version only
     assert len(_DataManager._get_all(version_number="latest")) == 2
     assert len(_TaskManager._get_all(version_number="latest")) == 1
-    assert len(_PipelineManager._get_all(version_number="latest")) == 1
+    assert len(_PipelineManager._get_all(version_number="all")) == 2
```

Comment:
```suggestion
    assert len(_PipelineManager._get_all(version_number="latest")) == 2
```
If `version_number="latest"`, how many number of pipeline entities exist?